medical_medicament_us/demo/medical_medicament_gcn_demo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="medical_medicament_gcn_us_1" model="medical.medicament.gcn">
+        <field name="name">Demo GCN</field>
```

Comment:
Is there anywhere where I can find GCN codes? Seems like most drugs just use NDC codes.